packages/unity-bootstrap-theme/src/scss/_end-style.scss

```diff
@@ -0,0 +1,93 @@
+
+/* -----------------------------------------------------
+Container/ row / column padding adjustments for mobile.
+
+Bootstrap doesn't natively provide a way to alter the behavior
+its native grid elements based on a media query. This overrides
+that behavior at screens <= 575px. (The small breakpoint.)
+
+Should be included after @import scss/grid. Registered here for clarity.
+------------------------------------------------------ */
+@mixin end_styles() {
+
+  @include media-breakpoint-down(md) {
+    .container,
+    .container-fluid,
+    .container-xl,
+    .container-lg,
+    .container-md,
+    .container-sm {
+      padding-left: 2rem;
+      padding-right: 2rem;
+    }
+  }
+
+
+  .container .container,
+  .container .container-sm,
+  .container .container-md,
+  .container .container-lg,
+  .container .container-xl,
+  .container .container-fluid,
```

Comment:
https://dev-webspark-release-stable.ws.asu.edu/
Header has nested `.container .container-fluid` is this a valid case?

Reply:
@scott-williams-az That should probably be corrected. Doesn't sound right.  Does this update trigger issues with that, or can it go forward and that issue be resolved after?

Reply:
WS2 would need to be corrected first